### PR TITLE
Module feedback analysis

### PR DESCRIPTION
**Pull request by Opeyem1a**

### CODE REVIEW

File: module_feedback_analysis.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+from scripts import analyze_module_feedback, compare_module_feedback
+
+MODULE_FEEDBACK_SURVEY_1_PATH = 'data/anonymized/ModuleFeedback1.csv'
+MODULE_FEEDBACK_SURVEY_2_PATH = 'data/anonymized/ModuleFeedback2.csv'
+
+if __name__ == '__main__':
+    module_feedback_survey_1_df = pd.read_csv(MODULE_FEEDBACK_SURVEY_1_PATH)
+    module_feedback_survey_2_df = pd.read_csv(MODULE_FEEDBACK_SURVEY_2_PATH)
+    # analyze_module_feedback(module_feedback_survey_1_df)
```
Comment (carson-ricca) [inline module_feedback_analysis.py:11]: Should this comment be here still?

File: scripts/analyze_module_feedback_survey/analyze_module_feedback_survey.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        return id_stripped_string
+
+    renamed_survey_df = survey_df.rename(remove_id_prefix, axis='columns')
+    return renamed_survey_df
+
+
+def preprocess_multiple_answer_data(data: Series) -> []:
+    separated_values = []
+    for value in data:
+        # FIXME: this could be better, honestly
```
Comment (carson-ricca) [inline scripts/analyze_module_feedback_survey/analyze_module_feedback_survey.py:65]: Definitely a little weird, could be worth looking into more?

File: scripts/analyze_module_feedback_survey/analyze_module_feedback_survey.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+def plot_bar_graph(data: iter, question_text: str):
+    set_plot_settings()
+    freq = create_answer_frequency_dict(data)
+
+    plt.bar(list(freq.keys()), list(freq.values()))
+    plt.xlabel("Answer Option")
+    plt.title("\n".join(wrap(f"'{question_text}'")))
+    # plt.subplots_adjust(top=0.65)
+    # plt.setp(plt.gca().get_xticklabels(), fontsize=10, rotation=45)
```
Comment (carson-ricca) [inline scripts/analyze_module_feedback_survey/analyze_module_feedback_survey.py:136]: Should these comments remain?

**Review by carson-ricca — CHANGES_REQUESTED** [review]
- Looks pretty good to me, just a couple of things I noticed.
- It may be a good idea to add some docstrings within the code to make it easier to understand.
- Updating the main readme with instructions on running it similar to the other scripts we have made is probably a good idea.

File: scripts/analyze_module_feedback_survey/analyze_module_feedback_survey.py (hunk 2 above)
Comment (Opeyem1a) [reply scripts/analyze_module_feedback_survey/analyze_module_feedback_survey.py:65]: I tried to improve it but tbh I couldn't devise a better way of handling it

**Review by carson-ricca — APPROVED** [review]

**Pull request merged**